image_width: 1920
image_height: 1080
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [1066.158339, 0.000000, 981.393777, 0.000000, 1068.659998, 545.569587, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.322670, 0.083221, 0.000164, -0.002134, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [741.315308, 0.000000, 968.865379, 0.000000, 0.000000, 969.430420, 546.524343, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
